lib/cf-socket.c

```diff
@@ -1158,13 +1158,13 @@ static CURLcode cf_tcp_connect(struct Curl_cfilter *cf,
 
     /* Connect TCP socket */
     rc = do_connect(cf, data, cf->conn->bits.tcp_fastopen);
-    set_local_ip(cf, data);
-    CURL_TRC_CF(data, cf, "local address %s port %d...",
-                ctx->l_ip, ctx->l_port);
     if(-1 == rc) {
       result = socket_connect_result(data, ctx->r_ip, SOCKERRNO);
       goto out;
     }
+    set_local_ip(cf, data);
+    CURL_TRC_CF(data, cf, "local address %s port %d...",
+                ctx->l_ip, ctx->l_port);
```

Comment:
When is this displayed?

Reply:
When `tcp` tracing is active. Do you want it on `infof()` instead?